app/controllers/project/conversations.js

```diff
@@ -2,6 +2,11 @@ import Controller from '@ember/controller';
 import { set } from '@ember/object';
 
 export default Controller.extend({
+  queryParams: ['status'],
+  status: 'open',
+
+  currentlyLoading: null,
```

Comment:
do you want the default to be false?

Reply:
I don't think so, since it could result in some weird state.